hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/PathMetadataDynamoDBTranslation.java

```diff
@@ -398,4 +398,15 @@ static String primaryKeyToString(PrimaryKey item) {
     }
     return "s3a://" + parent + "/" + child;
   }
+
+  /**
+   * Create an empty dir marker which, when passed to the
+   * DDB metastore, is considered authoritative.
+   * @param status file status
+   * @return path metadata.
+   */
+  static PathMetadata emptyDirectoryMarker(final S3AFileStatus status) {
```

Comment:
maybe change the method name to `emptyAuthoritativeDirectoryMarker`?

Reply:
done